PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py

```diff
@@ -159,8 +159,6 @@ def applySubstructure( process, postfix="" ) :
                     ]) ),
                     process, task)
 
-        
-    # Patify AK8 PF PUPPI
```

Comment:
the comment still applies apparently, but is it dropped now.
Not a big deal, but perhaps I missed something?

Reply:
will restore

Reply:
removed in https://github.com/cms-sw/cmssw/pull/19893/commits/a81f71edbe4501f3aae02326a93fef59d3ec7dc6